components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/test/java/org/wso2/carbon/identity/mgt/endpoint/util/RecoveryApiV2Test.java

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.mgt.endpoint.util;
+
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.MockitoAnnotations;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.ApiClient;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.ApiException;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.api.RecoveryApiV2;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2.AccountRecoveryType;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2.ConfirmRequest;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2.ConfirmResponse;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2.RecoveryInitRequest;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2.RecoveryRequest;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2.RecoveryResponse;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2.ResendRequest;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2.ResendResponse;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2.ResetRequest;
+import org.wso2.carbon.identity.mgt.endpoint.util.client.model.recovery.v2.ResetResponse;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+
+public class RecoveryApiV2Test {
+
+    public static final String RECOVERY_API_V2_RELATIVE_PATH = "/api/users/v2/recovery";
```

Comment:
make this private

Reply:
Fixed by https://github.com/wso2/carbon-identity-framework/pull/6040/commits/47a27ea6f9dea7d33e4630a1b7ce28f0fba5acb9